src/mme-app/actionHandlers/networkInitDetachActionHandlers.cpp

```diff
@@ -174,3 +174,32 @@ ActStatus ActionHandlers::process_ue_ctxt_rel_comp_for_detach(ControlBlock& cb)
 
 }
 
+/***************************************
+* Action handler : trigger_nwk_init_detach
+***************************************/
+ActStatus ActionHandlers::trigger_nwk_init_detach(ControlBlock& cb)
```

Comment:
PGW initiated detach is also network detach right ?

Reply:
Yes, removed this action and included ni_detach_req_to_ue and del_session_req.